lms/djangoapps/certificates/api.py

```diff
@@ -353,31 +352,30 @@ def cert_generation_enabled(course_key):
 
 
 def generate_example_certificates(course_key):
-    """Generate example certificates for a course.
+    """Generate example (PDF) certificates for a course.
 
-    Example certificates are used to validate that certificates
-    are configured correctly for the course.  Staff members can
-    view the example certificates before enabling
-    the self-generated certificates button for students.
+    Example certificates were used to validate that certificates were configured correctly for the course.  Staff
+    members could view the example certificates before enabling the self-generated certificates button for students.
 
-    Several example certificates may be generated for a course.
-    For example, if a course offers both verified and honor certificates,
-    examples of both types of certificate will be generated.
+    [07/20/2021 Update]
+    This function was updated to remove the references to queue.py, which has been removed as part of MICROBA-1227, and
+    no longer can fulfill the function it was originally created for. There is further cleanup around PDF certificate
+    generation code, part of DEPR-155, that will remove this function. See DEPR-155 and MICROBA-1094 for additional
+    info.
 
-    If an error occurs while starting the certificate generation
-    job, the errors will be recorded in the database and
-    can be retrieved using `example_certificate_status()`.
+    It may be important to note that this functionality has been broken since 2018 when the ability to generate PDF
+    certificates was ripped out of edx-platform. This will be removed as part of MICROBA-1394.
```

Comment:
MICROBA-1394 deals specifically with figuring out what to do with the ExampleCertificate model and related functions. MICROBA-1094 is a JIRA Epic containing all work related to removing PDF Certificate generation code. DEPR-155 is the DEPR ticket dealing with the removal of v1 course certificate generation functionality.